docs/builders/iso.mdx

```diff
@@ -82,6 +83,10 @@ in the image's Cloud-Init settings for provisioning.
 
 ### Optional:
 
+- `iso_packer_upload` (bool) - Download the specified `iso_url` first and
```

Comment:
Because this changes the default behaviour here, I would proceed cautiously at first, I don't know if this may be problematic, but just in case what would you think about reversing this condition so the default behaviour is still using Packer's Download/UploadISO steps?
Something like `iso_download_pve`?

Reply:
I guess you're right. Let's see if there arise any problems with the new method first.